@here/harp-mapview/lib/ConcurrentWorkerSet.ts

```diff
@@ -453,7 +467,11 @@ export class ConcurrentWorkerSet {
             } else {
                 logger.error(`[${this.m_options.scriptUrl}]: onWorkerMessage: invalid workerId`);
             }
-            entry.resolver(response.error, response.response);
+            if (response.error !== undefined) {
+                entry.resolver(new Error(response.error.toString()));
```

Comment:
I think `new Error(response.error.toString())` is not needed anymore as you have **instanceof check**

Reply:
It's needed because this `response.error` is `any` as defined in `WorkerServiceDefs.ts`. This decoder boundary, so we can't have "real" `Error` instances, only serializable forms which as for now are just results of `toString` applied on `Error` instances in workers.